flink-ml-parent/flink-ml-lib/src/main/java/org/apache/flink/ml/common/utils/RowTypeDataStream.java

```diff
@@ -0,0 +1,157 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.flink.ml.common.utils;
+
+import org.apache.flink.api.common.functions.MapFunction;
+import org.apache.flink.api.common.typeinfo.TypeInformation;
+import org.apache.flink.api.java.typeutils.RowTypeInfo;
+import org.apache.flink.ml.common.MLEnvironment;
+import org.apache.flink.ml.common.MLEnvironmentFactory;
+import org.apache.flink.streaming.api.datastream.DataStream;
+import org.apache.flink.table.api.Table;
+import org.apache.flink.table.api.TableSchema;
+import org.apache.flink.types.Row;
+
+/**
+ * Provide functions of conversions between DataStream and Table.
+ */
+public class RowTypeDataStream {
+	/**
+	 * Convert the given Table to {@link DataStream DataStream&lt;Row&gt;}.
+	 *
+	 * @param sessionId the sessionId of {@link MLEnvironmentFactory}
+	 * @param table the Table to convert.
+	 * @return the converted DataStream.
+	 */
+	public static DataStream <Row> fromTable(Long sessionId, Table table) {
```

Comment:
nit: If we rename these two APIs to `convertToDataStream` & `convertToDataSet` we can probably combine the two files. since `MLEnvironment` encapsulates both Batch and Stream envs. it is also consistent IMO.

I am fine with keeping two files and separate these APIs as well. maybe some other folks can provide some feedbacks on this.

Reply:
Thanks for your suggestion, I prefer to keep two files.